src/js/letters/utils/helpers.js

```diff
@@ -58,21 +37,169 @@ export const letterContent = {
+  hasIndividualUnemployabilityGranted: {
+    'true': {
+      veteran: <div>You are being paid at the 100 percent rate because you are unemployable due to your service-connected disabilities</div>,
+      dependent: undefined
+    },
+    'false': {
+      veteran: undefined,
+      dependent: undefined
+    }
+  },
+  hasSpecialMonthlyCompensation: {
+    'true': {
+      veteran: <div>You are service-connected for loss of or loss of use of a limb, or you are totally blind in or missing at least one eye</div>,
+      dependent: undefined
+    },
+    'false': {
+      veteran: undefined,
+      dependent: undefined
+    }
+  }
+};
+
+export function getBenefitOptionText(option, value, isVeteran) {
+  const personType = isVeteran ? 'veteran' : 'dependent';
+  let valueString;
+  if (value === false) {
+    valueString = 'false';
+  } else if (value === true) {
+    valueString = 'true';
+  } else {
+    valueString = value;
+  }
+
+  if (!['awardEffectiveDate', 'monthlyAwardAmount', 'serviceConnectedPercentage'].includes(option)) {
+    return benefitOptionText[option][valueString][personType];
+  }
+  switch (option) {
+    case 'awardEffectiveDate':
+      // TODO: verify that this value must be defined to show this option
+      if (value) {
+        return (<div>The effective date of the last change to your current award was <strong>{formatDateShort(value)}</strong></div>);
+      }
+      return undefined;
+    case 'monthlyAwardAmount':
+      if (value && value !== 'unavailable') {
+        return (<div>Your current monthly award amount is <strong>${value}</strong></div>);
+      }
+      return undefined;
+    case 'serviceConnectedPercentage':
+      if (value && value !== 'unavailable') {
+        if (isVeteran) {
+          return (<div>Your combined service-connected evaluation is <strong>{value}%</strong></div>);
+        }
+        return (<div>The Veteran's combined service-connected evaluation is <strong>{value}%</strong></div>);
+      }
+      break;
+    default:
+      return undefined;
+  }
+  return undefined;
 }
+
+// Lookup table to convert the benefit and military service options returned by the benefit summary letter
+// response to the expected request body options for customizing the benefit summary letter.
+export const benefitOptionsMap = {
+  hasAdaptedHousing: 'adaptedHousing',
+  hasChapter35Eligibility: 'chapter35Eligibility',
+  hasDeathResultOfDisability: 'deathResultOfDisability',
+  hasIndividualUnemployabilityGranted: 'unemployable',
+  hasNonServiceConnectedPension: 'nonServiceConnectedPension',
+  hasServiceConnectedDisabilities: 'serviceConnectedDisabilities',
+  hasSpecialMonthlyCompensation: 'specialMonthlyCompensation',
+  // A given user should only see one of these survivor award options and never both,
+  // so both map to the same request body option
+  hasSurvivorsIndemnityCompensationAward: 'survivorsAward',
+  hasSurvivorsPensionAward: 'survivorsAward',
+  awardEffectiveDate: 'monthlyAward',
```

Comment:
I don't think this one is right. From what Ian told me (and we might want to triple check this), `awardEffectiveDate` shouldn't appear as a clickable option in the list because it's not something they can optionally include in their letter or not. Therefore it doesn't map to anything that we send in the request for the pdf for bsl.